metricbeat/module/system/process_summary/process_summary.go

```diff
@@ -96,7 +96,7 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 		case 'Z':
 			summary.zombie++
 		default:
-			logp.Err("Unknown state <%v> for process with pid %d", state.State, pid)
+			logp.Err("Unknown or unexpected state <%c> (%s) for process with pid %d", state.State, prettyNameOf(state.State), pid)
```

Comment:
I agree with the change to print the character value, but I'd keep this error log line only for unknown values, and not for not-so-expected states as could be the dead one.